libr/anal/fcn.c

```diff
@@ -946,7 +947,7 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 		if (r_cons_is_breaked ()) {
 			break;
 		}
-		if ((len - addrbytes * idx) < 5) {
+		if ((len - addrbytes * idx) < 5 && len == MAXBBSIZE) { // Should use opt.bb_max_size here
```

Comment:
the maxbbsize thing should die

Reply:
added TODO bbd885b14e5bd1eb998c2c78068a051f6edb2880

Reply:
https://github.com/radare/radare2/issues/9564

Reply:
i'll have a look